run.py

```diff
@@ -0,0 +1,138 @@
+import os
+import re
```

Comment:
why is this run.py file here, there is one more in app folder, remove it if pushed by mistake

Reply:
The app folder is now removed, it was causing errors with imports so run.py is now in root dir